code/go/0chain.net/chaincore/chain/handler.go

```diff
@@ -1724,7 +1751,7 @@ func (c *Chain) TxnsInPoolHandler(w http.ResponseWriter, r *http.Request) {
 	// Print table and heading
 	fmt.Fprintf(w, "<table class='menu' cellspacing='10' style='border-collapse: collapse;'>")
 	fmt.Fprintf(w, "<th align='center' colspan='7'>Transactions in pool</th>")
```

Comment:
do not remove client id please, it is convenient to check nonce issues in the pool, most likely we should sort by it

Reply:
I'll add it back

Reply:
well, it's diagnostics, not sure we have to take of performance much, we can sort slice in memory